e2e-win/task_quoting.Tests.ps1

```
@@ -0,0 +1,57 @@
+
+Describe 'task inner-quote preservation (cmd /c)' {
+    # Regression tests for https://github.com/jdx/mise/discussions/9355:
+    # on Windows, inner double quotes in a task's `run` string were mangled when
+    # the default `cmd /c` shell was spawned, because the script was passed as an
+    # ordinary std::process::Command argument and re-quoted MSVCRT-style (inner
+    # `"` -> `\"`), which cmd.exe does not understand. The child then saw e.g.
+    # `\"hello world\"` instead of `"hello world"`, or `"import` instead of the
+    # whole `-c` argument. The fix passes the command to cmd verbatim.
+
+    BeforeAll {
+        $originalPath = Get-Location
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'originalPath' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)